com.trelby.Trelby.yaml

```diff
@@ -0,0 +1,33 @@
+app-id: com.trelby.Trelby
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: trelby
+finish-args:
+  - --device=all
```

Comment:
why do you need ALL devices?

Reply:
I'll remove that, sorry, I wasn't really sure how liberal I should be with permissions.